bin/picca_deltas.py

```diff
@@ -434,44 +488,46 @@ def main():
                                          log_file=log_file,
                                          keep_bal=args.keep_bal,
                                          bi_max=args.bi_max,
-                                         order=args.order,
                                          best_obs=args.best_obs,
                                          single_exp=args.single_exp,
                                          pk1d=args.delta_format,
                                          spall=args.spall)
 
```

Comment:
I'm not sure I understand the suggestion. best_obs, single_exp, spall are options for some of the read_from_* functions.

Reply:
What I meant is that you removed the order from the initilizer of Forest and as an argument for all the `read_from_*` routines, which I guess makes things less messy. But I'd suggest to still set the order in io.read_data(...), e.g. just before returning using the same 4 lines you've used here to set it. The reason would be to keep the picca_deltas cleaner as this doesn't actually do anything with the forests yet.

Reply:
I'm not sure what would be the best place for this. It is true that moving it to read_data(...) would keep picca_deltas cleaner, but adding it to read_data(...) feels weird in itself since the order is not actually read. I think this may be why Julian chose to move it here.

How about we add this as a method `set_order` of Forest? It would have the save 4 lines. I think this way we could have the benefits from all points of view

Reply:
This does sound like a good compromise to me. Another option that just came to my mind would be to have the data dictionary actually be an object to allow having a way to put all the 
```
for (heal)pix in data:
        for forest in data[(heal)pix]:
```
constructions into a function (or iterator or something), maybe even having parameters that are dataset and not really spectrum dependent such as e.g. order in there. But this might be too much of a change...

Reply:
Yes, I think it would be too much of a change

Reply:
In this case I'd opt for @iprafols suggestion to add a set_order method. Also: rename the `pix` in the for loops to healpix for consistency